mne/preprocessing/ica.py

```diff
@@ -144,36 +160,47 @@ class ICA(ContainsMixin):
     Parameters
     ----------
     n_components : int | float | None
-        The number of components used for ICA decomposition. If int, it must be
-        smaller than `max_pca_components`. If None, all PCA components will
-        be used. If float between 0 and 1, components will be selected by the
-        cumulative percentage of explained variance.
+        The number of PCA components from the pre-ICA PCA entering the ICA 
+        decomposition in the `fit()` method.
+        If int, it must be <= `max_pca_components`. 
+        If None, all PCA components will be used. If float between 0 and 1, 
+        components will be selected by the cumulative percentage of their 
+        explained variance.
     max_pca_components : int | None
-        The number of components used for PCA decomposition. If None, no
-        dimensionality reduction will be applied and `max_pca_components` will
-        equal the number of channels supplied for decomposing data.
+        The number of components returned by the PCA decomposition in the `fit()`
+        method. 
+        If None, no dimensionality reduction will be applied and `max_pca_components`
+        will equal the number of channels supplied for decomposing the data. 
+        If > `n_components_`, the additional PCA-only-components can later be used
+        for re-projecting the data into sensor space, additinally controllable 
+        by the `n_pca_components` parameter. It potentially [TO CHECK] 
+        also influences the choice of the actual PCA-algorithm used. Refer to the
+        docstring of sklearn.PCA() for details.
```

Comment:
Are you going to check this or are you asking somebody to do it?

Reply:
As mentioned in the PR description, it would be great if someone with some more serious background than me would check this. From what I understand, it does _not_ (any more) and the parameter could be removed completely, resp. merged with n_pca_components. But I would not rely on this.